src/network-analytics/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
```

Comment:
```suggestion
1.0.0b1
```
Adapt to the upcoming versioning strategy.

Reply:
version updated to 1.0.1